src/Refactoring-UI/StRefactoringPreviewPresenter.class.st

```diff
@@ -165,35 +165,35 @@ RBRefactoringPreviewPresenter >> initializePresenters [
 ]
```

Comment:
It is not necessary to call initialize the Layout by yourself. Now you should use `defaultLayout` method in which you need to return the desire layout, so the same layout that is built in `initializeLayout`

Reply:
Hey, thanks for the review! This is my first Spec window, so I was looking in multiple places to figure out how to create it and naturally I've copied some stuff that I shouldn't have. It makes much more sens to use `defaultLayout`, I've changed it.

Reply:
No problem, it is my pleasure! :)
The ìnitializeLayout`was a solution that we used to do but then the `defaultLayout` method was added! The chang is really recent.

Reply:
Oh, that's great, and the new way of defining layout is much better.